spec/support/factories.rb

```diff
@@ -0,0 +1,31 @@
+FactoryGirl.define do
+  factory :artist do
+    name
+    image_path "http://cps-static.rovicorp.com/3/JPG_400/MI0003/146/MI0003146038.jpg"
+  end
+
+  sequence :name do |n|
+    "#{n} Artist"
+  end
+
+  sequence :title, ["A", "C", "B"].cycle do |n|
+    "#{n} Title"
+  end
+
+  factory :song do
+    title
+    artist
+  end
+
+  sequence :playlist_name do |n|
+    "Playlist #{n} "
```

Comment:
Will try that!! Thank you!!!!

On Sunday, September 4, 2016, Sal Espinosa <notifications@github.com> wrote:

> In spec/support/factories.rb
> <https://github.com/susiirwin/mix-master/pull/2#discussion_r77458865>:
>
> > +
> > +  sequence :name do |n|
> > +    "#{n} Artist"
> > +  end
> > +
> > +  sequence :title, ["A", "C", "B"].cycle do |n|
> > +    "#{n} Title"
> > +  end
> > +
> > +  factory :song do
> > +    title
> > +    artist
> > +  end
> > +
> > +  sequence :playlist_name do |n|
> > +    "Playlist #{n} "
>
> Try to remove the trailing space here and let me know what happens.
>
> —
> You are receiving this because you authored the thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/susiirwin/mix-master/pull/2/files/5c7b982479552f814c9f29956409c6a81ddef6a7#r77458865>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/AQvABkq1Mkv8IWDNo7ozp57uvUXA8xiUks5qmzV9gaJpZM4J0m7V>
> .
>